### PR TITLE
Novacustom ns5x v1.2.0

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: docs/variants/novacustom_ns5x/releases.md [hunk 1]
```diff
@@ -14,6 +14,42 @@ For details about our release process please read
...
+## v1.2.0 - 2022-05-26
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=1695997523).
+
+### Added
+
+- Persistent RGB keyboard settings
+- Fix for CVE-2022-29264
+- Fix BIOS vendor name in SMBIOS
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/releases.md:26]: @mkopec If this says `fix`, it should be rather in a `fixed`  section? And be linked to an issue.

File: docs/variants/novacustom_ns5x/releases.md [hunk 2]
```diff
@@ -14,6 +14,42 @@ For details about our release process please read
...
+- Increased power limits
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/releases.md:27]: Can we elaborate a bit more? So end-user can have an ide what has changed

File: docs/variants/novacustom_ns5x/releases.md (hunk 1 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x/releases.md]: Moved and added the link

File: docs/variants/novacustom_ns5x/releases.md (hunk 2 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x/releases.md]: Added values for the new power limits

File: docs/variants/novacustom_ns5x/releases.md [hunk 3]
```diff
@@ -14,6 +14,45 @@ For details about our release process please read
...
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=1695997523).
+
+### Added
+
+- Persistent RGB keyboard settings
+- Increased power limits to CPU defaults (28W PL1 / 35W PL2)
+
+### Fixed
+
+- CVE-2022-29264 SMM loader vulnerability
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/releases.md:29]: We can link to e.g. https://nvd.nist.gov/vuln/detail/CVE-2022-29264 , can we?

File: docs/variants/novacustom_ns5x/releases.md [hunk 4]
```diff
@@ -14,6 +14,45 @@ For details about our release process please read
...
 [here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit?usp=sharing).
 
+## v1.2.0 - 2022-05-26
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=1695997523).
+
+### Added
+
+- Persistent RGB keyboard settings
```
Comment (macpijan) [inline docs/variants/novacustom_ns5x/releases.md:24]: There was no public issue or documentation about that request I assume?

File: docs/variants/novacustom_ns5x/releases.md (hunk 3 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x/releases.md]: Added

File: docs/variants/novacustom_ns5x/releases.md (hunk 4 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x/releases.md:24]: No, no public mention of it

**Comment by macpijan** [comment]
@mkopec pre-commit does not pass

**Comment by mkopec** [comment]
@macpijan Fixed. Sorry, pre-commit is still broken on my workstation

**Review by macpijan — APPROVED** [review]

**Pull request merged**